core/src/software/aws/toolkits/core/region/Partitions.kt

```diff
@@ -48,16 +47,10 @@ object PartitionParser {
         mapper.readValue(inputStream, Partitions::class.java)
     }
 }
-object EndpointsJsonValidator : RemoteResolveParser {
```

Comment:
Should this be deleted? Would this object be required to validate the endpoints file?